### PR TITLE
BI-27: Test drive Search Bar

### PR DESCRIPTION
**Pull request by ayushtiwari110**

### CODE REVIEW

**Review by bacpactech — APPROVED** [review]

File: src/__tests__/home.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import Home from '../app/page.js'
+import SearchBar from '@/components/SearchBar.js'
+import '@testing-library/jest-dom'
+ 
+describe('Home', () => {
+  it('renders a heading', () => {
+    render(<Home />)
+ 
+    const heading = screen.getByRole('heading', {
+      name: /BacPac/i,
```
[the remark this answers is not in the document]
Comment (ayushtiwari110) [reply src/__tests__/home.test.js:11]: sure!

File: src/__tests__/searchBar.test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import data from '../../data/university_data'
+import '@testing-library/jest-dom'
+
+it('renders a search box', () => {
+  const {getByPlaceholderText} = render(<SearchBar data={data}/>)
+  
+  const subject = getByPlaceholderText('Search Colleges...')
+
+  expect(subject).toBeInTheDocument()
+})
```
[the remark this answers is not in the document]
Comment (ayushtiwari110) [reply src/__tests__/searchBar.test.js:12]: done!

File: src/__tests__/searchBar.test.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+it('display filtered data based on input', () => {
+  const {getByPlaceholderText, queryByText} = render(<SearchBar data={data}/>)
+  const input = getByPlaceholderText('Search Colleges...')
+  
+  fireEvent.change(input, { target: { value: 'Bhubaneswar' } })
+  // console.log(screen.debug()); //For Debugging: Print the rendered component to the console
```
[the remark this answers is not in the document]
Comment (ayushtiwari110) [reply src/__tests__/searchBar.test.js:19]: done!

File: .prettierignore [hunk 4]
```diff
@@ -2,8 +2,15 @@
 .eslintrc.json
 .prettierrc.json
 .storybook/
+/src
```
Comment (bacpactech) [inline .prettierignore:5]: Let's remove the `/src`.

**Review by bacpactech — CHANGES_REQUESTED** [review]

File: .prettierignore (hunk 4 above)
Comment (ayushtiwari110) [reply .prettierignore:5]: done!

**Review by bacpactech — DISMISSED** [review]
Nicely Done! 🎉

**Review by bacpactech — APPROVED** [review]

**Pull request merged**